src/sugar3/graphics/toolbutton.py

```diff
@@ -180,5 +233,9 @@ def do_draw(self, cr):
         return False
 
     def do_clicked(self):
+        '''
+        Implementation method for hiding the tooltip when the
+        tool button is clicked
```

Comment:
Some of your paragraphs end in a period (.) and some do not.  Please be consistent.